### DISCUSSION TITLE
Logical Advice

### DISCUSSION THREAD

**Discussion by CICABERR** [seleniumbase/SeleniumBase#2927] [Q&A] [answered]
Hello,

First of all, I really congratulate you on your work and success. You have truly produced a very successful work.

My question is as follows: I am using SeleniumBase to scrape data from a website. Then I need to log into another website to use this data. If I open the website I need to log into in another tab in the same browser, will it increase my chances of getting caught by bot detectors like Cloudflare and Google reCAPTCHA on the first and second websites? What usage recommendations do you suggest?

Secondly, there are 20 links on the website I am scraping data from, and each link has 50 pages. To save time, if I start the scraping process for all 20 links simultaneously using the same code structure with different file names on the same computer, will it increase my chances of getting caught by bot detectors like Cloudflare and Google reCAPTCHA?

Thank you.

**Comment by mdmintz** [comment, marked as answer]
Thank you. For a lot of that, you may need to experiment in order to determine what works best. If you're opening a new tab in your tests, you'll need to make sure that you're using the UC Mode methods for that, as a opening a tab via the regular Selenium way won't disconnect the driver from the browser, which is needed for evading Selenium-detection.

I think some of the existing UC Mode examples can help you with test design. See [SeleniumBase/help_docs/uc_mode.md](https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/uc_mode.md).
